### PR TITLE
DDF-1648 fixing compile and runtime issues

### PR DESCRIPTION
**Pull request by stustison**
@pklinef @coyotesqrl 

@vinamartin can hero

### CODE REVIEW

File: platform/admin/ui/package.json [hunk 1]
```diff
@@ -13,7 +13,7 @@
     "url": "https://github.com/codice/ddf.git"
   },
   "engines": {
-    "node": "5.5.0"
+    "node": ">=0.10.5"
```
Comment (coyotesqrl) [inline platform/admin/ui/package.json]: Why is this the inverse of the change to `admin-metrics-ui/package.json`?

File: pom.xml [hunk 2]
```diff
@@ -56,8 +56,8 @@
         <!-- ouput directory for reports -->
         <project.report.output.directory>project-info</project.report.output.directory>
         <!-- Properties for the frontend-maven-plugin -->
-        <node.version>v0.10.40</node.version>
-        <npm.version>2.14.8</npm.version>
+        <node.version>v4.3.1</node.version>
```
Comment (coyotesqrl) [inline pom.xml]: This version shouldn't match the 5.5?

File: platform/admin/ui/package.json (hunk 1 above)
Comment (stustison) [inline platform/admin/ui/package.json]: I forgot to change it back.

File: pom.xml (hunk 2 above)
Comment (stustison) [inline pom.xml]: This is the LTS version of node and is the recommended version to use.

**Comment by pklinef** [comment]
:+1:

**Pull request merged**